### PR TITLE
Port to jbuilder

### PR DESCRIPTION
**Pull request by euanh**

### CODE REVIEW

File: lib/passwd.ml [hunk 1]
```diff
@@ -119,8 +118,8 @@ let to_string p =
     p.shell
 
 let db_to_string db = db
-  |> List.map to_string
-  |> String.concat "\n"
+                      |> List.map to_string
```
Comment (lindig) [inline lib/passwd.ml]: This looks strange. Is the file mixing tabs and spaces?

File: lib/shadow.ml [hunk 2]
```diff
@@ -126,8 +125,8 @@ let to_string p =
     p.flag
 
 let db_to_string db = db
-  |> List.map to_string
-  |> String.concat "\n"
+                      |> List.map to_string
```
Comment (lindig) [inline lib/shadow.ml]: Same as above. I's suggest a `ocp-indent` run over this file.

Comment (euanh) [inline lib/shadow.ml]: I did run ocp-indent - that's what made it this way! :/

Comment (euanh) [inline lib/shadow.ml]: It doesn't like the `db` after the `=` in the function declaration.   If I add a newline after the `=` it does something more sensible.

**Comment by lindig** [comment]
Why did Travis fail? It would not show me the logs - I triggered a new build.

File: _oasis [hunk 3]
```diff
@@ -1,41 +0,0 @@
...
-Name:        passwd
-Version:     1.0.1
-Synopsis:    OCaml interface to Unix password library
-Authors:     Mike McClurg
-License:     ISC
-LicenseFile: LICENSE
-Plugins:     META (0.4), DevFiles(0.4)
-BuildTools:  ocamlbuild
-
-Flag extra_debugging
```
Comment (euanh) [inline _oasis]: Need to put something equivalent to this debug flag back in befor merging.

Comment (lindig) [inline _oasis]: Can we get this or should we close this PR until we can get it?

**Comment by robhoes** [comment]
We'll merge this now. Any further tweaks can be done later.

**Pull request merged**